README.md

```diff
@@ -98,8 +98,6 @@ require 'hanami/mailer'
 User = Struct.new(:name, :username)
 luca = User.new('Luca', 'jodosha')
 
-Hanami::Mailer.load!
```

Comment:
I've removed this line because I didn't think it's contextual. Some point about it?